src/main/java/frc/team3388/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,51 @@
+package frc.team3388.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.flash3388.flashlib.robot.scheduling.Subsystem;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.SpeedController;
+import frc.team3388.robot.EncoderSrx;
+
+public class Shooter extends Subsystem {
+    private WPI_TalonSRX speedController;
```

Comment:
Whenever possible you should make things more generic, so instead of using `WPI_TalonSRX` you should use the `SpeedController` class in case we will want to use other controllers.

Reply:
it was speedcontroller before i just had to change it to wpi_talonsrx to use the PID stuff

Reply:
oh right, makes sense